app/controllers/members_controller.rb

```diff
@@ -34,11 +46,11 @@ def toggle_active
   private
 
   def find_member
-    @member ||= current_organization.members.find(params[:id])
+    @member ||= @organization.members.find(params[:id])
```

Comment:
Uff this repo is full of this kind of stuff :(

Yep, will do. but maybe once I finish to move the resources under the `/organizations` scope.